src/libs/ReportUtils.ts

```diff
@@ -2394,7 +2394,7 @@ function getIcons(
 
         // For one transaction IOUs, display a simplified report icon
         if (isOneTransactionReport(report?.reportID ?? '-1')) {
```

Comment:
https://github.com/Expensify/App/blob/803821640f120e5674a8b307e701b4799c1897c0/src/libs/ReportUtils.ts#L1664

@grgia I think we should also fix this line to avoid falling into here when there're expenses from multiple users. Note `undefined` is not equal to `null`, so it should be 
```javascript
    return !!ReportActionsUtils.getOneTransactionThreadReportID(reportID, reportActions);
```
Below is the recording to compare my end with the adhoc build


https://github.com/user-attachments/assets/9f2fc866-ed4b-48b3-8517-92592b22497b




See also
1. The design https://github.com/Expensify/App/issues/49036#issuecomment-2348655371

Reply:
Nice catch, Updated!